examples/config.hocon.sample

```diff
@@ -0,0 +1,111 @@
+# Default configuration for s3-loader
+
+# Sources currently supported are:
+# 'kinesis' for reading records from a Kinesis stream
+# 'nsq' for reading records from a NSQ topic
+source: "{{source}}"
+
+# Sink is used for sending events which processing is failed.
+# Sinks currently supported are:
+# 'kinesis' for writing records to a Kinesis stream
+# 'nsq' for writing records to a NSQ topic
+sink: "{{sink}}"
+
+# The following are used to authenticate for the Amazon Kinesis sink.
+#
+# If both are set to 'default', the default provider chain is used
+# (see http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html)
+#
+# If both are set to 'iam', use AWS IAM Roles to provision credentials.
+#
+# If both are set to 'env', use environment variables AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
+aws {
+  accessKey: "iam"
+  secretKey: "iam"
+}
+
+# Config for NSQ
+nsq {
+  # Channel name for NSQ source
+  channelName: "{{nsqSourceChannelName}}"
+    
+  # Host name for NSQ tools
+  host: "{{nsqHost}}"
+
+  # Port for nsqd
+  port: "{{nsqdPort}}"
+
+  # Port for nsqlookupd
+  lookupPort: {{nsqlookupdPort}}
+}
+
+kinesis {
+  # LATEST: most recent data.
+  # TRIM_HORIZON: oldest available data.
+  # "AT_TIMESTAMP": Start from the record at or after the specified timestamp
+  # Note: This only affects the first run of this application on a stream.
+  initialPosition: "{{kinesisInitialPosition}}"
+
+  # Need to be specified when initialPosition is "AT_TIMESTAMP".
+  # Timestamp format need to be in "yyyy-MM-ddTHH:mm:ssZ".
+  # Ex: "2017-05-17T10:00:00Z"
+  # Note: Time need to specified in UTC.
+  initialTimestamp: "{{timestamp}}"
+
+  # Maximum number of records to read per GetRecords call     
+  maxRecords: {{kinesisMaxRecords}}
+
+  region: "{{kinesisRegion}}"
+
+  # "appName" is used for a DynamoDB table to maintain stream state.
+  # You can set it automatically using: "SnowplowLzoS3Sink-$\\{sink.kinesis.in.stream-name\\}"
```

Comment:
it's "SnowplowLzoS3Sink-$\\{streams.inStreamName\\}" now

Reply:
on second thought let's remove this entirely, this isn't really a good idea as people won't want to change it if they create a new stream

Reply:
let's remove it in the es-loader too if present